packages/dds/shared-summary-block/src/test/sharedSummaryBlock.spec.ts

```diff
@@ -75,7 +76,7 @@ describe("SharedSummaryBlock", () => {
             const value3 = { value: "testValue3" };
             sharedSummaryBlock.set(key3, value3);
 
-            const tree = sharedSummaryBlock.snapshot();
+            const tree = (sharedSummaryBlock as SharedSummaryBlock).snapshot();
```

Comment:
I assume e need to change that soon, as we will remove snapshot, right? Do we have an issue tracking it?

Reply:
This is next in my list along with returning the GC nodes from summarize() in SharedObject.
I created a task for this - #4431